src/internal/components/chart-plot/index.tsx

```diff
@@ -167,7 +167,8 @@ function ChartPlot(
   return (
     <>
       <svg
-        {...restProps}
```

Comment:
Remove destructured props, only use explicit ones for better clarity. `onFocus` and `onBlur` were being assigned twice but it was not easy to tell because of this.